homeassistant/components/minio/__init__.py

```diff
@@ -136,8 +137,7 @@ def put_file(service: ServiceCall) -> None:
         file_path = _render_service_value(service, ATTR_FILE_PATH)
 
         if not hass.config.is_allowed_path(file_path):
-            _LOGGER.error("Invalid file_path %s", file_path)
-            return
+            raise HomeAssistantError(f"Invalid file_path {file_path}")
```

Comment:
At the moment we normally raise `ValueError` in service handlers if the problem is bad user input. If it's eg an api call failure we raise `HomeAssistantError`.

Reply:
I've tried with a different error .. but had to then look through logs, to see what the actual issue is
Raising `HomeAssistantError` actually shows the error message on the UI

but again, if you want me to change it, I don't care ..

Reply:
That's probably a consequence of the specific frontend solution for the way the service call is executed. The message may eg show if you wrap the call in a script in an automation when testing it.

We should stick with our standard here so it's easier for us to find these cases when we refactor to a more specific exception, which may also lead to a better UX.